css/layouts/responsive.css

```diff
@@ -80,6 +80,58 @@
   }
 }
 
+/* Footer ABN and Tagline */
+@media all and (min-width: 580px) {
+  /* The footer abn and tagline */
+  .region-footer .block {
+    width: 50%;
+  }
+  .region-footer .menu::after {
+    position: relative;
+    float: left;
+    clear: both;
+    display: block;
+    content: "ABN 63 384 330 717";
```

Comment:
Yep, I agree. This is definitely content that should be output as HTML, not added in at the style layer. With these kind of decisions, you always need to assume things like CSS pseudo elements *won't* be supported.